pages/knowledge/colours-and-contrast/text/README.md

```diff
@@ -0,0 +1,79 @@
+
+For a more direct comparison: the following image shows both large and default text with contrast ratios of exactly `3:1` and `4.5:1`.
+
+![Small and large text](_media/small-and-large-text.png)
+
+It may not seem like much of a difference, but in fact it heavily enlargens the number of possible colour combinations.
+
+It is important to mention that the values `3:1` and `4.5:1` represent a *minimal* standard for *acceptable* contrast, they don’t represent an ideal. They should be treated as threshold and not as target; aiming higher is recommended. 
+
+## Enhancing text contrast
+
+Besides changing the colour of the text or the background there are many other ways do enhance contrast. For example:
+
+### Adding shadow
+
+Drop shadow around text can make up for insufficient contrast between text colour and background.
+
+For example, look at the following image. It shows two words, both with a red fill that has an insufficient contrast of `2.1:1` to the white background.
+
+![A word without and a word with shadow](_media/words-without-and-with-shadow.png)
+
+The shadow of the second word results in a contrast of appropriately `4.5:1`.
+
+### Adding a contrast layer
+
+In some cases, text is displayed on non-homogeneous backgrounds. Depending on the heterogeneity of the background, the text may be very hard to read.
+
+For example, look at the following image. It shows a white text “Welcome to the beach!” on blue sky with some white clouds. While there is decent contrast between white text and blue sky there is almost no contrast between white text and white clouds. Overall it’s very hard to read.
+
+![White text on blue sky and white clouds](_media/beach.png)
+
+You could increase contrast by adding a text shadow as described above. Or you could add a semi-transparent darkening layer between text and background
+
+![White text with semi-transparent darkening background](_media/beach-with-background.png)
+
+Of course you can also combine different solutions to add up their effects.
+
+## Exceptions
+
+There are some exceptions to the contrast requirements of text content.
+
+- **Logos** usually must adhere strictly to corporate design guidelines and thus may be displayed in those colours regardless of contrast ratios (needless to say that it still is unpleasant if your clients cannot perceive your logo).
+- **Decorative text:** this could be the background pattern of an image that is created from random words.
+- **Incidental text:** this could be a random street sign somewhere in the background of a photograph.
+- **Disabled text:** this could be the label of a disabled element in an online form.
+
+## “Real” text vs. text images
+
+With “real” text, we mean text that is available as an arrangement of character codes (like this text here). It is inherently machine-readable and can be selected and copy-pasted. “Text images” on the other hand are rasterised pictures of typography, e.g. stored as JPG or PNG files. 
```

Comment:
I would add " (like the example images in the text above)" at the end of the sentence.